agent/config/testdata/full-config.hcl

```diff
@@ -306,6 +306,11 @@ limits {
     rpc_max_conns_per_client = 2954
     kv_max_value_size = 1234567800
     txn_max_req_len = 567800000
+    request_limits {
+        mode = "permissive"
+        read_rate = 100.0
+        write_rate = 100.0
```

Comment:
Maybe worth making these different values to catch any conversion typos?

Reply:
Done.  🙏